README.md

```diff
@@ -109,13 +100,33 @@ Finished: SUCCESS
 The exit code reflects the result of the build. The `test` directory of this workspace includes a very simple
 example of Jenkinsfile that can be used to demo Jenkinsfile Runner.
 
-### Passing parameters
+### CLI options
+The executable of Jenkinsfile Runner allows its invocation with these cli options:
 
-Any parameter values, for parameters defined on workflow job within `parameters` statement
-can be passed to the Jenkinsfile Runner using `-a` or `--arg` switches in key=value format. 
+```
+ # Usage: jenkinsfile-runner
+ --runWorkspace FILE     : Path to the workspace of the run to be used within
```

Comment:
It might make sense to add a short handle like `-ws` @varyvol 
Not a big deal tho